module/Photo/src/Photo/Controller/PhotoController.php

```diff
@@ -14,8 +14,35 @@ public function indexAction()
         $albums = $album_service->getAlbums();
         $photo_service = $this->getPhotoService();
 
+        //we'll fix this ugly thing later vv
+        $config = $this->getPhotoService()->getConfig();
+        $basedir = str_replace("public", "", $config['upload_dir']);
+        $albums = $photo_service->populateCoverPhotos($albums);
```

Comment:
populateCoverPhotos() checkt voor elke album in de array of er een cover photo is ingesteld, als dit niet zo is wordt er een random cover photo gekozen. Deze wordt niet opgeslagen in de database, zodat er elke keer een andere foto wordt weergegeven. (Zoals het nu ook is op de huidige website)